samples/ldscripts/armv4t.ld

```diff
@@ -0,0 +1,7 @@
+__flash =      0x00000000;
```

Comment:
These appear to be the same in all 3 new ld files - can this be generalized to provide one file? or do you expect them to be different in the future?

BTW, does this specify a specific board? Maybe then the file can be named after the board like we have for micro:bit and Raspberry Pi.

Reply:
I've followed the convention from the picolibc test patch which landed today. Currently there are 9 other identical files in that folder. @pwprzybyla and @mplatings would be in a better position to answer.

Reply:
Hm, now I see them, though the question is do they belong more to the tests folder, than samples? that way it would be clear that they follow the config files naming - test config for every library config.

Reply:
@voltur01 I'm fine if you want to move the files around in a different change. For now, I think where Ties has put the files is appropriate.

Reply:
I agree that it is a possible refactoring on the testing side, not related to this patch, so OK with me.